src/components/Game.jsx

```diff
@@ -40,8 +35,8 @@ export class Game extends React.Component {
     }
   }
 
-  continueClickHandler = () => {
-    const {rounds, updateRound, history, fetchPeople} = this.props;
+  let continueClickHandler = () => {
+    const {rounds, updateRound, history, fetchPeople} = props;
```

Comment:
This can be removed if you decide to go with destructuring things in the params of the Game component